test/vm.install

```diff
@@ -12,7 +12,7 @@ lvextend -r -l +100%FREE $VG/root
 
 # overriding osbuild-composer repo with nightly
 mkdir -p /etc/osbuild-composer/repositories
-cp /home/admin/files/rhel-8-beta.json /etc/osbuild-composer/repositories/
+cp /home/admin/files/rhel-8-beta.json /etc/osbuild-composer/repositories/rhel-8.json
```

Comment:
Thanks! I'll land this now to unblock testing -- but should it be renamed in the source as well?

Reply:
Or perhaps, copy it as both names, so that it still continues to work once we move to 8.4 alpha/beta?